docs/source/3x/transformers_like_api.md

```diff
@@ -0,0 +1,236 @@
+from neural_compressor.transformers import AutoModelForCausalLM, AutoRoundConfig
+
+model_name_or_path = "MODEL_NAME_OR_PATH"
+tokenizer = AutoTokenizer.from_pretrained(model_name_or_path)
+woq_config = AutoRoundConfig(bits=4, tokenizer=tokenizer)
+woq_model = AutoModelForCausalLM.from_pretrained(
+    model_name_or_path,
+    quantization_config=woq_config,
+)
+
+# inference
+from transformers import AutoTokenizer
+
+prompt = "Once upon a time, a little girl"
+tokenizer = AutoTokenizer.from_pretrained(model_name_or_path)
+
+input_ids = tokenizer(prompt, return_tensors="pt")["input_ids"]
+generate_kwargs = dict(do_sample=False, temperature=0.9, num_beams=4)
+gen_ids = q_model.generate(input_ids, **generate_kwargs)
+gen_text = tokenizer.batch_decode(gen_ids, skip_special_tokens=True)
+print(gen_text)
+```
+
+You can also save and load your quantized low bit model by the below code.
+
+```python
+# quant
+from neural_compressor.transformers import AutoModelForCausalLM, RtnConfig
+
+model_name_or_path = "MODEL_NAME_OR_PATH"
+woq_config = RtnConfig(bits=4)
+q_model = AutoModelForCausalLM.from_pretrained(
+    model_name_or_path,
+    quantization_config=woq_config,
+)
+
+# save quant model
+saved_dir = "SAVE_DIR"
+q_model.save_pretrained(saved_dir)
+
+# load quant model
+loaded_model = AutoModelForCausalLM.from_pretrained(saved_dir)
+```
+
+## Usage For Intel GPU
+Intel® Neural Compressor implement weight-only quantization for intel GPU(PVC/ARC/MTL) with [Intel-extension-for-pytorch](https://github.com/intel/intel-extension-for-pytorch).
+
+Now 4-bit/8-bit inference with `RtnConfig`, `GPTQConfig`, `AutoRoundConfig` are support on intel GPU device.
+
+We support experimental woq inference on intel GPU(PVC/ARC/MTL) with replacing Linear op in PyTorch. Validated models: Qwen-7B, Llama-7B, Phi-3.  
+
+Here are the example codes.
+
+#### Prepare Dependency Packages
+1. Install Oneapi Package  
+The Oneapi DPCPP compiler is required to compile intel-extension-for-pytorch. Please follow [the link](https://www.intel.com/content/www/us/en/developer/articles/guide/installation-guide-for-oneapi-toolkits.html) to install the OneAPI to "/opt/intel folder".
+
+2. Build and Install PyTorch and Intel-extension-for-pytorch
+```python
+python -m pip install torch==2.3.1+cxx11.abi --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/
+
+# Build IPEX from Source Code
+git clone https://github.com/intel/intel-extension-for-pytorch.git ipex-gpu
+cd ipex-gpu
+git submodule update --init --recursive
+export USE_AOT_DEVLIST='pvc,ats-m150'  # Comment this line if you are compiling for MTL
+export BUILD_WITH_CPU=OFF
+export LD_LIBRARY_PATH=${CONDA_PREFIX}/lib/:$LD_LIBRARY_PATH
+export OCL_ICD_VENDORS=/etc/OpenCL/vendors
+export CCL_ROOT=${CONDA_PREFIX}
+source /opt/intel/oneapi/setvars.sh --force
+export LLM_ACC_TEST=1
+pip install -r requirements.txt
+
+python setup.py install
+```
+
+3. Install Neural-compressor
```

Comment:
This is inside INC doc,  no need for install INC section

Reply:
removed, lll comments have been resolved